web/modules/custom/pipeline_log/pipeline_log.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Pipeline log
```

Comment:
We can leave `pipeline_log` as module directory, but let's follow the rule of prefixing with `joinup_*` the module machine-name: `joinup_pipeline_log`. However, from my perspective, we don't need a new module, we have `joinup_federation`. But this is up on you.